subprojects/docs/src/main/resources/header.html

```diff
@@ -167,10 +167,10 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                     <li><a href="../userguide/plugins.html">Using Gradle Plugins</a></li>
                     <li><a href="../userguide/build_lifecycle.html">Understanding the Build Lifecycle</a></li>
                     <li><a href="../userguide/logging.html">Working with Logging</a></li>
-                    <li><a href="../userguide/multi_project_builds.html">Configuring Multi-Project Builds</a></li>
                     <li><a href="../userguide/potential_traps.html">Avoiding Traps</a></li>
                 </ul>
             </li>
+            <li><a href="../userguide/multi_project_builds.html">Multi-Project Builds</a></li>
```

Comment:
:o: When we updated the structure of the _dependency management_ part of the user manual, we made all titles follow the "What you do"-wording - use case driven if you want. I think we should do the same for the whole user manual, as we update different parts.

Here we could go with:

`Authoring Multi-Project Builds` if that is not too long

Or maybe something less technical like just

`Structuring Builds`

I think something like this might work if we get the additional menu items below when we split up the chapter into pieces. We can also rename this again then.

Reply:
Sounds good. I will change it to `Authoring Multi-Project Builds` for this change to keep it consistent with the rest.
When/if the further restructuring happens, then I think it would make sense to rename it to `Structuring Builds` where one sub-section would specifically focus on `Creating a multi-project build`.